### PR TITLE
Cleanup custom-columns CLI flag

### LINKED ISSUE #1526: Cleanup custom-columns CLI flag
## Current situation

We used to have a CLI flag `custom-columns` but it does not exist anymore: it was renamed to `columns`. However, `git grep custom-columns` shows left overs in comments, documentation, tools and code.

`custom-columns` was renamed to `columns` by #1281.

## Impact

Suggested commands from the code base actually don't work anymore:
```
$ ./kubectl-gadget trace dns -n demo --timeout 3 -o custom-columns=pod,comm,qr,qtype,name
Error: invalid output mode "custom-columns"
```

## Ideal future situation

`git grep custom-columns` does not give any results.

## Implementation options

Remove `custom-columns` or rename as appropriate.

## Additional information

```
$ git grep custom-columns
cmd/common/utils/flags.go:30:   OutputModeCustomColumns = "custom-columns"
docs/gadgets/audit/seccomp.md:65:$ kubectl gadget audit seccomp -o custom-columns=namespace,pod,syscall,code
docs/gadgets/common-features.md:50:This can be overridden with either `json` or `custom-columns`.
docs/gadgets/common-features.md:78:Using `-o custom-columns=column1,column2` we can choose which columns to
docs/gadgets/common-features.md:87:$ kubectl gadget trace oomkill -A -o custom-columns=kpid,kcomm
docs/gadgets/profile/cpu.md:89:$ kubectl gadget profile cpu --timeout 1 --podname random -o custom-columns=node,pod
docs/gadgets/profile/cpu.md:100:$ kubectl gadget profile cpu --timeout 1 --podname random -o custom-columns=node,namespace,pod,container,pid,comm,count,stack
docs/gadgets/top/ebpf.md:59:you can call the gadget with the custom-columns option and specify the cumulruntime and cumulruncount columns.
docs/gadgets/top/ebpf.md:64:$ kubectl-gadget top ebpf -o custom-columns=node,progid,type,name,pid,comm,cumulruntime,cumulruncount --sort cumulruntime --timeout 60
docs/gadgets/trace/capabilities.md:165:    -o custom-columns=comm,syscall,capName,verdict,targetuserns,currentuserns,caps,capsnames
docs/ig.md:113:- JSON format and `custom-columns` output mode are supported through the
integration/command.go:209:               NAMESPACES=$(kubectl get seccompprofile --all-namespaces --no-headers --ignore-not-found -o custom-columns=":metadata.namespace" | uniq)
pkg/gadgets/traceloop/types/types.go:79:        // -o custom-columns.
tools/demos/trace-dns/demo.sh:24:run "kubectl gadget trace dns -n demo --timeout 3 -o custom-columns=pod,comm,qr,qtype,name"
tools/demos/trace-exec/demo.sh:21:run "kubectl gadget trace exec -n demo --timeout 4 -o custom-columns=pod,pid,comm,args"
```

### PR DESCRIPTION
**Pull request by shivaabhishek07**
# Cleanup custom-columns CLI flag

This PR fixes #1526 as the `custom-columns` flag has been changed to `columns`, the changes in this PR will replace all the `custom-columns` flag from documentation and code with `columns`

### CODE REVIEW

**Comment by shivaabhishek07** [comment]
Yes @mauriciovasquezbernal , moving the constants from registry.go to flags.go will remove the duplicate code. I will work on it.
And I will also change `OutputModeCustomColumns` in flags.go to `OutputModeColumns`.

File: cmd/common/registry.go
Comment (mauriciovasquezbernal) [inline cmd/common/registry.go:50]: Please removed commented code.

File: cmd/common/utils/flags.go
Comment (mauriciovasquezbernal) [inline cmd/common/utils/flags.go:33]: same here.

**Review by mauriciovasquezbernal — COMMENTED** [review]
It's looking very good. Would you mind squashing the two commits together?

File: integration/command.go
Comment (mauriciovasquezbernal) [inline integration/command.go:1]: This command is not part of kubectl-gadget, so custom-columns **has** to be used in this case.

File: tools/demos/trace-exec/demo.sh
Comment (mauriciovasquezbernal) [inline tools/demos/trace-exec/demo.sh:1]: Please follow the instructions on tools/demos/README.md to update the generated image in docs/builtin-gadgets/trace/exec.gif. (same for trace dns)

**Review by mauriciovasquezbernal — COMMENTED** [review]
I think it's almost ready to go, just a couple of comments. 

Would you mind also editing the commit message to avoid having a lot of `Signed-off-by` there?

Once you update it, I'll trigger the CI and merge it everything is green.

File: tools/demos/trace-exec/demo.sh
Comment (shivaabhishek07) [reply tools/demos/trace-exec/demo.sh:1]: For this, I need help from @alban because I don't have the same interface as shown in the existing demos. If I record and change this, it looks different from other gif files.

**Comment by mauriciovasquezbernal** [comment]
@shivaabhishek07 there are a lot of failures on the CI (lint, build, etc), would you mind fixing them?

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM but since I made some changes I'd like to get a review from somebody else.

**Review by burak-ok — COMMENTED** [review]
There is still an occurence of `custom-columns`:
```
./integration/command.go:		  NAMESPACES=$(kubectl get seccompprofile --all-namespaces --no-headers --ignore-not-found -o custom-columns=":metadata.namespace" | uniq)
```

I don't know if the following needs a change:
```
./pkg/columns/doc.go:	| hide      | none                   | specifies that this column is not to be considered by default (see custom columns) 
```

**Comment by mauriciovasquezbernal** [comment]
> There is still an occurence of `custom-columns`:
> 
> ```
> ./integration/command.go:		  NAMESPACES=$(kubectl get seccompprofile --all-namespaces --no-headers --ignore-not-found -o custom-columns=":metadata.namespace" | uniq)
> ```

That one is from kubectl, not kubectl-gadget.

> I don't know if the following needs a change:

Updated it.

**Comment by burak-ok** [comment]
> > There is still an occurence of `custom-columns`:
> > ```
> > ./integration/command.go:		  NAMESPACES=$(kubectl get seccompprofile --all-namespaces --no-headers --ignore-not-found -o custom-columns=":metadata.namespace" | uniq)
> > ```
> 
> That one is from kubectl, not kubectl-gadget.

Went too fast :D

**Review by burak-ok — APPROVED** [review]
LGTM

**Pull request merged**
